Ansible/group_vars/identity-ethernet

```diff
@@ -0,0 +1,36 @@
+---
+# Certificate information
+#  which CA issued the certificate
+cert_CA: comodo
+# this is the list of the root and intermediate certificates of the CA
+# should probably give these keys names...
+cert_CA_certificates:
+  - COMODORSADomainValidationSecureServerCA.crt
+  - AddTrustExternalCARoot.crt
+  - COMODORSAAddTrustCA.crt
+# institute metadata
+site_name: Catch-All
```

Comment:
You should select a different name for your site, e.g.
`site_name: EtherNet`

The site_name variable is used  to pick up things in `group_vars` so setting it to "Catch-All" will pick up catch-all variables. Catch-All is actually a site :smile: (sorry, that's a bit confusing!)